packages/types/src/index.ts

```diff
@@ -241,6 +241,11 @@ export interface PlainTextInput extends Action {
   multiline?: boolean;
   min_length?: number;
   max_length?: number;
+  dispatch_action_config?: DispatchActionConfig;
```

Comment:
https://api.slack.com/reference/block-kit/block-elements#input is missing if `dispatch_action_config` is required or not. I'm assuming it isn't required.

Reply:
Good catch. We should probably report this to the api.slack.com team as well

Reply:
Yup! I reported it to them!

Reply:
>I'm assuming it isn't required.

This field is surely optional.